### PR TITLE
Issue fixes, changes & new features

### PR DESCRIPTION
**Pull request by corecompete10028**
**Master template**
VPCCIDR Parameter passed to SASGrid Stack

**sas-grid template**
Added VPCCIDR Parameter.
Nested stack order changed from SASGrid, Lustre to Lustre, SASGrid

**Lustre template**
Removed unwanted parameters & resources

1. Parameter (EFSFileSystemId)
2. Removed EFS Filesytem & Mount Target resources

Added new regions & resources

1. IAM Roles & Policies for DynamoDB entries and update tags of resources
2. Added new regions
3. NFS
4. Success Signalling for EC2 resources and Custom resources.
5. Fixed mounting issues and reboot order.
6. Added resources like WaitCondition & WaitConditionHandle for Custom Resources Signalling
7. EC2 instances launch with IAM Roles attached
8. Outputs of 3 resources

**sas-nodes template**
Remove unwanted parameters & resources

1. Parameter (EFSFileSystemId)
2. Removed EFS Filesytem & Mount Target resources

Added new regions & resources

1. IAM Roles & Policies for DynamoDB entries and update tags of resources
2. Added new regions
3. NFS
4. Success Signalling for EC2 resources and Custom resources.
5. Fixed mounting issues and reboot order.
6. Added resources like WaitCondition & WaitConditionHandle for Custom Resources Signalling
7. EC2 instances launch with IAM Roles attached
8. Outputs of 3 resources

### CODE REVIEW

File: templates/lustre.template [hunk 1]
```diff
@@ -209,6 +215,104 @@
...
+                        {
+                            "Sid": "Stmt1527587282041",
+                            "Action": [
+                                "dynamodb:DescribeTable",
+                                "dynamodb:GetItem",
+                                "dynamodb:GetRecords",
+                                "dynamodb:PutItem"
+                            ],
+                            "Effect": "Allow",
+                            "Resource": "*"
```
Comment (vsnyc) [inline templates/lustre.template]: This should be restricted to just the ServerList table. Read/Write access to '*' is too broad and not desired.

File: templates/lustre.template [hunk 2]
```diff
@@ -242,9 +346,18 @@
...
+                                        "dynamodb:GetItem",
+                                        "dynamodb:Scan",
+                                        "dynamodb:UpdateItem",
+                                        "dynamodb:UpdateTable",
+                                        "iam:PassRole",
+                                        "iam:ListAttachedRolePolicies",
+                                        "iam:AttachRolePolicy"
                                     ],
                                     "Resource": [
                                         "*"
```
Comment (vsnyc) [inline templates/lustre.template]: Please split this into multiple statements and restrict the resource '*' for dynamodb to the ServersList table.

File: templates/lustre.template (hunk 1 above)
Comment (corecompete10028) [reply templates/lustre.template]: This change is completed.

File: templates/lustre.template (hunk 2 above)
Comment (corecompete10028) [reply templates/lustre.template]: I have removed "dynamodb" actions, as we don't need anymore for this lambda function.

**Pull request merged**